src/ontology/mondo-edit.obo

```diff
@@ -491794,15 +491779,29 @@ intersection_of: has_characteristic HP:0011463 ! Childhood onset
 [Term]
 id: MONDO:0037251
 name: congestive splenomegaly
+def: "A chronic congestive enlargement of the spleen leading to the destruction of blood cells resulting in pancytopenia." [PMID:35800794]
 subset: inferred_rare
+subset: nord_rare {source="MONDO:NORD"}
 subset: rare
-synonym: "banti syndrome" RELATED []
-synonym: "banti's spleen" RELATED []
-synonym: "congestive splenomegaly" EXACT []
-synonym: "fibrocongestive splenomegaly" RELATED []
+synonym: "Banti syndrome" EXACT [MONDO:0021969]
+synonym: "Banti's disease" EXACT [GARD:0005888, MESH:C537903]
+synonym: "Banti's spleen" EXACT [https://orcid.org/0000-0001-5208-3432]
+synonym: "Banti's syndrome" EXACT [GARD:0005888]
+synonym: "congestive splenomegaly" EXACT [https://orcid.org/0000-0001-5208-3432]
```

Comment:
I think this can be removed as it is the same as the label name and does not have a "preferred label" tag.

Reply:
done